visualizer/style/semanticsEditor.css

```diff
@@ -209,3 +217,32 @@ headerBlock real {
   color: #999;
 }
 
+/* `Force Evaluation` entry in `parseTreeMenu` */
+.contextMenu label span {
+  margin-left: 10px;
+}
+.contextMenu li.disabled ul {
+  display: none;
+}
+.contextMenu ul ul {
+  left: 100%;
+  top: 0;
+  padding: 0;
+  position: absolute;
+  border-radius: 4px;
```

Comment:
Can you keep these in alphabetical order? I like to do it that way, since otherwise the order tends to get a bit random. It's fine that some CSS doesn't use alphabetical order, but let's keep the stuff that is already ordered consistent.